styles/OpenLoanStyles.ts

```diff
@@ -32,4 +32,6 @@ export const OpenLoanStyles = {
   confirmButtonContainer: 'pb-4 w-full',
   interestRateText: 'w-full text-right text-gray-200 pr-4',
   spacer: 'p-5',
+  balanceContainer: 'flex',
```

Comment:
Please remove style changes.

Reply:
This property is added by me because of the div that I made. So, that's why I need this style changes. It does not change/effect any prior element's design.